data/drop-tables.js

```diff
@@ -11,6 +11,7 @@ async function run() {
     await client.query(`
             DROP TABLE IF EXISTS users CASCADE;
             DROP TABLE IF EXISTS animals;
```

Comment:
You can get rid of the animals data and references throughout if you'd like. Not required, but cleans things up a bit